src/share/util/shr_scam_mod.F90

```diff
@@ -935,8 +957,26 @@ subroutine get_close( targetlon, targetlat, nlon, lons, nlat, lats, closelonidx,
 
    !--- find index of value closest to 0 and set returned values ---
 
-   closelonidx=(MINLOC(abs(poslons-postargetlon),dim=1))
-   closelatidx=(MINLOC(abs(lats-targetlat),dim=1))
+   !--- if SCM uses Eulerian grids
+   if (nlat .ne. nlon) then
```

Comment:
Again I don't think this is the best test to use to determine what grid you are on.